Error when importing server jar "sizes don't match"

When opening a server jar with the latest Enigma version (0.14.2.143), it crashes with this stack trace:
`java.lang.Error: java.lang.RuntimeException: org.objectweb.asm.tree.analysis.AnalyzerException: Error at instruction 117: sizes don't match
	at cuchaz.enigma.gui.dialog.ProgressDialog.lambda$runOffThread$0(ProgressDialog.java:70)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.RuntimeException: org.objectweb.asm.tree.analysis.AnalyzerException: Error at instruction 117: sizes don't match
	at cuchaz.enigma.analysis.index.IndexReferenceVisitor.lambda$visitMethod$0(IndexReferenceVisitor.java:49)
	at cuchaz.enigma.analysis.MethodNodeWithAction.visitEnd(MethodNodeWithAction.java:17)
	at org.objectweb.asm.ClassReader.readMethod(ClassReader.java:1287)
	at org.objectweb.asm.ClassReader.accept(ClassReader.java:688)
	at org.objectweb.asm.ClassReader.accept(ClassReader.java:400)
	at cuchaz.enigma.analysis.ClassCache.visit(ClassCache.java:108)
	at cuchaz.enigma.analysis.index.JarIndex.indexJar(JarIndex.java:67)
	at cuchaz.enigma.analysis.ClassCache.index(ClassCache.java:123)
	at cuchaz.enigma.Enigma.openJar(Enigma.java:55)
	at cuchaz.enigma.gui.GuiController.lambda$openJar$1(GuiController.java:90)
	at cuchaz.enigma.gui.dialog.ProgressDialog.lambda$runOffThread$0(ProgressDialog.java:66)
	... 1 more
Caused by: org.objectweb.asm.tree.analysis.AnalyzerException: Error at instruction 117: sizes don't match
	at org.objectweb.asm.tree.analysis.Analyzer.analyze(Analyzer.java:295)
	at cuchaz.enigma.analysis.index.IndexReferenceVisitor.lambda$visitMethod$0(IndexReferenceVisitor.java:47)
	... 11 more
Caused by: java.lang.IllegalStateException: sizes don't match
	at cuchaz.enigma.analysis.InterpreterPair.pair(InterpreterPair.java:100)
	at cuchaz.enigma.analysis.InterpreterPair.merge(InterpreterPair.java:88)
	at cuchaz.enigma.analysis.InterpreterPair.merge(InterpreterPair.java:14)
	at org.objectweb.asm.tree.analysis.Frame.merge(Frame.java:676)
	at org.objectweb.asm.tree.analysis.Analyzer.merge(Analyzer.java:541)
	at org.objectweb.asm.tree.analysis.Analyzer.analyze(Analyzer.java:267)
	... 12 more
`
Any idea how to fix this problem?
The server jar is: https://launcher.mojang.com/v1/objects/0d0d7db69eb40408c23ce4503dfd00671bcb9e6b/server.jar

Also having this issue with the 1.16 server snapshot jar;

`java.lang.Error: java.lang.RuntimeException: org.objectweb.asm.tree.analysis.AnalyzerException: Error at instruction 117: sizes don't match
	at cuchaz.enigma.gui.dialog.ProgressDialog.lambda$runOffThread$0(ProgressDialog.java:70)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.RuntimeException: org.objectweb.asm.tree.analysis.AnalyzerException: Error at instruction 117: sizes don't match
	at cuchaz.enigma.analysis.index.IndexReferenceVisitor.lambda$visitMethod$0(IndexReferenceVisitor.java:49)
	at cuchaz.enigma.analysis.MethodNodeWithAction.visitEnd(MethodNodeWithAction.java:17)
	at org.objectweb.asm.ClassReader.readMethod(ClassReader.java:1287)
	at org.objectweb.asm.ClassReader.accept(ClassReader.java:688)
	at org.objectweb.asm.ClassReader.accept(ClassReader.java:400)
	at cuchaz.enigma.analysis.ClassCache.visit(ClassCache.java:108)
	at cuchaz.enigma.analysis.index.JarIndex.indexJar(JarIndex.java:67)
	at cuchaz.enigma.analysis.ClassCache.index(ClassCache.java:123)
	at cuchaz.enigma.Enigma.openJar(Enigma.java:55)
	at cuchaz.enigma.gui.GuiController.lambda$openJar$1(GuiController.java:90)
	at cuchaz.enigma.gui.dialog.ProgressDialog.lambda$runOffThread$0(ProgressDialog.java:66)
	... 1 more
Caused by: org.objectweb.asm.tree.analysis.AnalyzerException: Error at instruction 117: sizes don't match
	at org.objectweb.asm.tree.analysis.Analyzer.analyze(Analyzer.java:295)
	at cuchaz.enigma.analysis.index.IndexReferenceVisitor.lambda$visitMethod$0(IndexReferenceVisitor.java:47)
	... 11 more
Caused by: java.lang.IllegalStateException: sizes don't match
	at cuchaz.enigma.analysis.InterpreterPair.pair(InterpreterPair.java:100)
	at cuchaz.enigma.analysis.InterpreterPair.merge(InterpreterPair.java:88)
	at cuchaz.enigma.analysis.InterpreterPair.merge(InterpreterPair.java:14)
	at org.objectweb.asm.tree.analysis.Frame.merge(Frame.java:676)
	at org.objectweb.asm.tree.analysis.Analyzer.merge(Analyzer.java:541)
	at org.objectweb.asm.tree.analysis.Analyzer.analyze(Analyzer.java:267)
	... 12 more
`

Unfortunately, it stills happen with the latest version, even when using CFR :/
This is the stacktrace with Procyon:
```
java.lang.Error: java.lang.RuntimeException: org.objectweb.asm.tree.analysis.AnalyzerException: Error at instruction 117: sizes don't match
	at cuchaz.enigma.gui.dialog.ProgressDialog.lambda$0(ProgressDialog.java:71)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.RuntimeException: org.objectweb.asm.tree.analysis.AnalyzerException: Error at instruction 117: sizes don't match
	at cuchaz.enigma.analysis.index.IndexReferenceVisitor.lambda$0(IndexReferenceVisitor.java:49)
	at cuchaz.enigma.analysis.MethodNodeWithAction.visitEnd(MethodNodeWithAction.java:17)
	at org.objectweb.asm.ClassReader.readMethod(ClassReader.java:1287)
	at org.objectweb.asm.ClassReader.accept(ClassReader.java:688)
	at org.objectweb.asm.ClassReader.accept(ClassReader.java:400)
	at cuchaz.enigma.analysis.ClassCache.visit(ClassCache.java:108)
	at cuchaz.enigma.analysis.index.JarIndex.indexJar(JarIndex.java:69)
	at cuchaz.enigma.analysis.ClassCache.index(ClassCache.java:123)
	at cuchaz.enigma.Enigma.openJar(Enigma.java:55)
	at cuchaz.enigma.gui.GuiController.lambda$0(GuiController.java:97)
	at cuchaz.enigma.gui.dialog.ProgressDialog.lambda$0(ProgressDialog.java:67)
	... 1 more
Caused by: org.objectweb.asm.tree.analysis.AnalyzerException: Error at instruction 117: sizes don't match
	at org.objectweb.asm.tree.analysis.Analyzer.analyze(Analyzer.java:295)
	at cuchaz.enigma.analysis.index.IndexReferenceVisitor.lambda$0(IndexReferenceVisitor.java:47)
	... 11 more
Caused by: java.lang.IllegalStateException: sizes don't match
	at cuchaz.enigma.analysis.InterpreterPair.pair(InterpreterPair.java:100)
	at cuchaz.enigma.analysis.InterpreterPair.merge(InterpreterPair.java:88)
	at cuchaz.enigma.analysis.InterpreterPair.merge(InterpreterPair.java:1)
	at org.objectweb.asm.tree.analysis.Frame.merge(Frame.java:676)
	at org.objectweb.asm.tree.analysis.Analyzer.merge(Analyzer.java:541)
	at org.objectweb.asm.tree.analysis.Analyzer.analyze(Analyzer.java:267)
	... 12 more
```
This seems to be like a pain in the ass to fix :(

--- a/src/main/java/cuchaz/enigma/analysis/InterpreterPair.java
+++ b/src/main/java/cuchaz/enigma/analysis/InterpreterPair.java
@@ -97,10 +97,6 @@ public class InterpreterPair<V extends Value, W extends Value> extends Interpret
             return null;
         }
 
-        if (left != null && right != null && left.getSize() != right.getSize()) {
-            throw new IllegalStateException("sizes don't match");
-        }
-
         return new PairValue<>(left, right);
     }
 
@@ -111,10 +107,6 @@ public class InterpreterPair<V extends Value, W extends Value> extends Interpret
         public PairValue(V left, W right) {
             if (left == null && right == null) {
                 throw new IllegalArgumentException("should use null rather than pair of nulls");
-            }
-
-            if (left != null && right != null && left.getSize() != right.getSize()) {
-                throw new IllegalArgumentException("sizes don't match");
             }
 
             this.left = left;

